packit-ci.fmf

```diff
@@ -0,0 +1,51 @@
+summary: run ansible system roles test with IMA policy setup
+
+environment:
+  DOCKERFILE_AGENT: Dockerfile.agent
+  DOCKERFILE_SYSTEMD: Dockerfile.systemd
+
+context:
+  swtpm: no
+  agent: rust
+
+provision:
+  hardware:
+    tpm:
+      version: "2"
+    virtualization:
+      is-virtualized: true
+
+prepare:
+  - how: shell
+    script:
+     - rpm -Uv https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm https://dl.fedoraproject.org/pub/epel/epel-next-release-latest-9.noarch.rpm || true
+    when: distro == rhel-9 or distro == centos-stream-9
+  - how: shell
+    script:
+     - ln -s $(pwd) /var/tmp/keylime_sources
```

Comment:
@sergio-correia I am sorry, the name of the link should be `/var/tmp/keylime_server-role-sources`.